test/crawler_web/views/home_view_test.exs

```diff
@@ -0,0 +1,39 @@
+defmodule CrawlerWeb.HomeViewTest do
+  use CrawlerWeb.ConnCase, async: true
+
+  alias CrawlerWeb.HomeView
+
+  describe "get_column_status/2" do
+    test "given status new, returns text-info",
+      do: assert(HomeView.get_column_status(:new) == "text-info")
+
+    test "given status in_progress, returns text-info",
+      do: assert(HomeView.get_column_status(:in_progress) == "text-warning")
+
+    test "given status completed, returns text-info",
+      do: assert(HomeView.get_column_status(:completed) == "text-success")
+
+    test "given status failed, returns text-info",
+      do: assert(HomeView.get_column_status(:failed) == "text-danger")
+
+    test "given status undefined, returns text-info",
+      do: assert(HomeView.get_column_status(:undefined) == "text-muted")
+  end
+
+  describe "get_status/2" do
+    test "given status new, returns text-info",
```

Comment:
```suggestion
    test "given status new, returns New",
```

Same for the other test 💥

Reply:
aw, good point 🙏 Fixed in [e2b3fed](https://github.com/vnntsu/elixir-ic/pull/40/commits/e2b3fedf187e39859db833210730fb6f9f9215a1)